sdk/translation/azure-ai-translation-document/azure/ai/translation/document/models/__init__.py

```diff
@@ -6,13 +6,49 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
+from ._models import BatchRequest
+from ._models import DocumentFilter
+from ._patch import DocumentStatus
 from ._models import DocumentTranslateContent
-from ._patch import __all__ as _patch_all
-from ._patch import *  # pylint: disable=unused-wildcard-import
+from ._models import DocumentTranslationFileFormat
+from ._patch import TranslationGlossary
```

Comment:
Has this been manually edited?

Reply:
I tried re-generating the SDK and there were no changes in this file, so NO, it was not manually edited